tests/functional/spec/features/anchor_scrolling.spec.js

```diff
@@ -0,0 +1,63 @@
+import InsuranceAddressPage from "../../generated_pages/section_summary/insurance-address.page.js";
+import InsuranceTypePage from "../../generated_pages/section_summary/insurance-type.page.js";
+import ListedPage from "../../generated_pages/section_summary/listed.page.js";
+import PropertyDetailsSummaryPage from "../../generated_pages/section_summary/property-details-section-summary.page.js";
+import HouseType from "../../generated_pages/section_summary/house-type.page.js";
+import HouseholdCountSectionSummaryPage from "../../generated_pages/section_summary/household-count-section-summary.page.js";
+import HouseholdDetailsSummaryPage from "../../generated_pages/section_summary/house-details-section-summary.page.js";
+import NumberOfPeoplePage from "../../generated_pages/section_summary/number-of-people.page.js";
+import SubmitPage from "../../generated_pages/section_summary/submit.page.js";
+
+describe("Section Summary", () => {
+  describe("Given I start a Test Section Summary survey", () => {
+    beforeEach(() => {
+      browser.openQuestionnaire("test_section_summary.json");
+      $(InsuranceTypePage.both()).click();
+      $(InsuranceTypePage.submit()).click();
+    });
+
+    it("When I have provided an answer and click through to the next question, the Previous button url shouldn't yet contain any anchors or reference to return_to or return_to_answer_id", () => {
```

Comment:
We usually use the pattern Given / When / Then, so these statements need a tweak